entrypoint.sh

```diff
@@ -60,6 +74,7 @@ elif [ "$CI" = "true" ] && [ -n "$BITBUCKET_BUILD_NUMBER" ]; then
   head_branch="${BITBUCKET_BRANCH}"
   export INPUT_REPORTER='bitbucket-code-report'
   export INPUT_FILTER_MODE='nofilter'
+  triggering_actor="$BITBUCKET_STEP_TRIGGERER_UUID"
```

Comment:
How much work is it to get the PR creator for Bitbucket?

Reply:
2 story points .

Reply:
Tracked at https://ekline.atlassian.net/browse/EK-607